packages/block-editor/src/components/block-canvas/index.js

```diff
@@ -38,7 +38,7 @@ export function ExperimentalBlockCanvas( {
 			>
 				<EditorStyles
 					styles={ styles }
-					scope=".editor-styles-wrapper"
+					scope="div:where(.editor-styles-wrapper)"
```

Comment:
```suggestion
					scope=":where(.editor-styles-wrapper)"
```

This selector doesn't match when the iframe preview is active (class is applied to body, not a div). We can just remove the tag from the selector and it will work just as well.

Reply:
The selector doesn't need to match the iframed state, because this instance of `EditorStyles` only renders when there's no iframe. 

[This comment](https://github.com/WordPress/gutenberg/pull/56649#issuecomment-2017165803) explains why the increased specificity provided by the `div` selector is necessary.

Reply:
Could we add a comment inline?

Reply:
When testing in 6.5, this selector was still being applied even with the iframe preview. It seemed like the intent of this PR was to ensure it's always available no matter if the iframe preview is in use or not. In it's current form, this will break if it applies to an iframe, since the iframe markup has this class on the body and not a div. 

If the intended behavior is for this to only show without an iframe preview, then it appears there's a bug somewhere. I replicated this yesterday with this branch checkout and the latest 6.5 RC (4, I believe).

Reply:
Comment added!

> this selector was still being applied even with the iframe preview

Do you mean `.editor-styles-wrapper` or `div:where(.editor-styles-wrapper)`? If the latter, could you please provide detailed reproduction steps? The change in this PR is exclusively targeting the non-iframed editor. However, `.editor-styles-wrapper` may still be in use in the iframed editor.